docs/ContributorHub/open-positions-and-bounties.md

```diff
@@ -10,166 +10,233 @@ nav_order: 1
+
+- A DAO/DeFi enthusiast who believes in the power of decentralization
+- An independent, self-starter who is comfortable working in a fast-paced environment
+- Someone who has helped build brands and tell unique stories in the past
+- Proven experience in successful marketing in the Asia market
+- Someone to help grow our presence throughout various online communities and find creative ways to drive high levels of interest in our brand within the Asia market
+- Experience managing marketing campaigns and working with internal and third-party resources to execute
+
+# Product Support Manager
 
 ___
 
-DXdao is a decentralized collective native to the internet that is building a suite of Web3 products. As such, it heavily relies on the technical expertise of Web3 developers with solid knowledge of building dApps on top of Ethereum.
+DXdao is a decentralized collective native to the internet that is building a suite of Web3 products.  As these products grow, product management and support is critical to the success of DXdao and its products.  DXdao is looking to empower an enthusiastic **Product Support Manager** who wants to jump right in and start contributing to important initiatives, while interacting with all parts of DXdao.
 
-**What you should be interested in doing**:
 
--   Interested in DeFi
-    
--   Interested in solving complex problems (a background in finance/STEM helps here)
-    
--   A constant mindset for refactoring software
+### What you should be interested in doing:
 
-**DXdao is looking for**:
+- Working on the cutting edge of decentralization
+- Contributing to a distributed, consensus based, onchain organization
+- Driving the execution of all lifecycle processes for DeFi products, including product research, market research, competitive analysis, planning, positioning, roadmap development, requirements development, and product launch
+- Translating product strategy into detailed requirements and prototypes
 
--   A DAO/DeFi enthusiast who believes in the power of decentralization
-    
--   Solid experience in React and its ecosystem
-    
--   Solid understanding of dApps Architecture
-    
--   Comfortable with Solidity language
-    
--   Write mainatable, high quality code
-    
--   Knowledge of Docker, Jest, and IPFS
-    
--   Able to work in Agile environment (Scrum)
-    
--   Implements SOLID principles of SWE
-    
--   Self-starter and entrepreneurship spirit are a huge plus!
+### DXdao is looking for:
 
-# Biz Dev Whiz
+- A DAO/DeFi enthusiast who believes in the power of decentralization
+- An independent, self-starter who is comfortable working in a fast-paced environment
+- A contributor with extensive experience in product management/support
+- Experience with Ethereum or other EVM compatible blockchains; preferably in DeFi or decentralized governance
+- Someone with solid experience in security and testing
+- A contributor with excellent written and verbal communication skills 
+
+
+# QA Engineer
```

Comment:
@KeenanLukeOM I think here it makes good sense to use table of contents and the top for every open positions so people can have overview of positions available without having to scroll down...

Reply:
Good idea! We should have a chat about it, I was tinkering with a TOC but it kept including subtext. Can give example in a bit.